static/app/views/issueDetails/groupEventCarousel.tsx

```diff
@@ -508,11 +485,11 @@ const ActionsWrapper = styled('div')`
   gap: ${space(0.5)};
 `;
 
-const StyledNavButton = styled(Button)`
+export const StyledNavButton = styled(Button)`
```

Comment:
TBH I'd rather just copy/paste these styles into `eventDisplay.tsx`. The components in this file weren't meant to be reused in other contexts - if we want to make a reusuable component we should create a proper abstraction and put it in the top level components folder